Netlify build function failed

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
Build function successfully

## Current Behavior
Build function failed

## Possible Solution
Add netlify.tolm under dashboard folder

<!--- Provide a general summary of the issue in the Title above -->

## Detailed Description and log
10:00:38 PM: No netlify.toml found. This is needed to configure the function settings. For more info: https://github.com/netlify/netlify-lambda#installation
10:00:38 PM: npm ERR! code ELIFECYCLE
10:00:38 PM: npm ERR! errno 1
10:00:38 PM: npm
10:00:38 PM: ERR! cello-dashboard@1.0.0 functions:build: `npm run generateMock && netlify-lambda build ./lambda`
10:00:38 PM: npm
10:00:38 PM: ERR! Exit status 1
10:00:38 PM: npm
10:00:38 PM:  ERR!
10:00:38 PM: npm
10:00:38 PM:  ERR!
10:00:38 PM:  Failed at the cello-dashboard@1.0.0 functions:build script.
10:00:38 PM: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
10:00:38 PM: npm
10:00:38 PM:  ERR! A complete log of this run can be found in:
10:00:38 PM: npm ERR!     /opt/buildhome/.npm/_logs/2019-11-21T14_00_38_805Z-debug.log
10:00:38 PM: Function Dir: /opt/build/repo/src/dashboard/functions

## Possible Implementation
<!--- Not obligatory, but suggest an idea for implementing addition or change -->

--- a/src/dashboard/netlify.toml
+++ b/src/dashboard/netlify.toml
@@ -1,0 +1,2 @@
+[build]
+  functions = "functions"

